packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -657,7 +657,7 @@ export class ContextualMenu extends BaseComponent<IContextualMenuProps, IContext
 
   private _getIconProps(item: IContextualMenuItem): IIconProps {
     const iconProps: IIconProps = item.iconProps ? item.iconProps : {
-      iconName: item.icon
+      iconName: 'None'
```

Comment:
is this function even necessary anymore? thought it was only a helper to get the icon from item.icon or item.iconProps.iconName, depending on what was available.

Reply:
It does still seem useful in that it defaults to 'None' if the item has no iconProps.